python/README.md

```diff
@@ -0,0 +1,56 @@
+# This folder contains manual bindings for aikidopy using pybind11.
+
+0. The installation guide have been tested on
+
+- sudo apt installed python 2.7 and python 3.4.
+- anaconda managed python 3.6
+
+If you use system installed python, ensure you have `python2.7-dev` or `python3-dev` correspondingly, or run `sudo apt-get install python3-dev` to obtain it.
+
+
+1. Install [pybind11](https://github.com/pybind/pybind11.git) **from source** following this [instruction](https://pybind11.readthedocs.io/en/master/basics.html#compiling-the-test-cases). (Need version >= 2.2.0).
+
+```
+git clone https://github.com/pybind/pybind11.git
```

Comment:
Bash commands should be prepended with a `$`, Python commands should be prepended with `>>>`.